tests/core/mempool/test_mempool_manager.py

```diff
@@ -257,6 +263,22 @@ async def generate_and_add_spendbundle(
     return (sb, sb_name, result)
 
 
+def mempool_item_from_spendbundle(spend_bundle: SpendBundle) -> MempoolItem:
+    generator = simple_solution_generator(spend_bundle)
+    npc_result = get_name_puzzle_conditions(
+        generator=generator, max_cost=INFINITE_COST, cost_per_byte=COST_PER_BYTE, mempool_mode=True
```

Comment:
is `INFINITE_COST` important here? I would prefer some reasonable limit

Reply:
Done.